### ISSUE TITLE
imdb not working

### ISSUE THREAD

**Issue by alexxandeer** [swetorrentking/rartracker#67]
When typing a imdb URL in the upload form the field next to the URL that are suppose to show the MovieTitle gives me Error: [object Object]. 
Console gives me : Failed to load resource: the server responded with a status of 500 (Internal Server Error) api/v1/moviedata/imdb/tt1386697 

Do anyone know whats the problem

**Comment by conyum** [comment]
Did you solve it? :)
If so, post you fix so that others can use it to :)

**Comment by alexxandeer** [comment]
Replace akas to www in the imdb url's in MovieData.php

**Comment by ERR1R** [comment]
https://github.com/ERR1R/rartracker

**Comment by alexxandeer** [comment]
Any plans to fix for the new imdb again? @ERR1R

**Comment by ERR1R** [comment]
Do you use my fork?

**Comment by alexxandeer** [comment]
Yes i do, but it seems that imdb has changed their source again.

**Comment by ERR1R** [comment]
ok, i  pushed some changes.. Better to make comments on the fork :D
ps. i also removed swe stuff and subs.. Also, added imagick to resize imdb images 500 width..
ps2..  one thing is broken and that is the watch button.. dunno why yet..

**Comment by alexxandeer** [comment]
Can I find you on Wire or any other message service?
